mercury/forms.py

```diff
@@ -133,7 +133,73 @@ class Meta:
             "oil_decrease_rate",
             "created_at",
         ]
-
+        widgets = {
+            "name": forms.TextInput(
+                attrs={"id": "post-name", "required": True, "placeholder": "Enter Name"}
+            ),
+            "owner": forms.TextInput(
+                attrs={
+                    "id": "post-owner",
+                    "required": True,
+                    "placeholder": "Enter Name of the owner",
+                }
+            ),
+            "temperature": forms.NumberInput(
+                attrs={"id": "post-temperature", "required": True}
+            ),
+            "acceleration_x": forms.NumberInput(
+                attrs={"id": "post-acceleration-X", "required": True}
+            ),
+            "acceleration_y": forms.NumberInput(
+                attrs={"id": "post-acceleration-Y", "required": True}
+            ),
+            "acceleration_z": forms.NumberInput(
+                attrs={"id": "post-acceleration-Z", "required": True}
+            ),
+            "wheel_speed_fr": forms.NumberInput(
+                attrs={"id": "post-wheel-speed-fr", "required": True}
+            ),
+            "wheel_speed_fl": forms.NumberInput(
+                attrs={"id": "post-wheel-speed-fl", "required": True}
+            ),
+            "wheel_speed_br": forms.NumberInput(
+                attrs={"id": "post-wheel-speed-br", "required": True}
+            ),
+            "wheel_speed_bl": forms.NumberInput(
+                attrs={"id": "post-wheel-speed-bl", "required": True}
+            ),
+            "suspension_fr": forms.NumberInput(
+                attrs={"id": "post-suspension-fr", "required": True}
+            ),
+            "suspension_fl": forms.NumberInput(
+                attrs={"id": "post-suspension-fl", "required": True}
+            ),
+            "suspension_br": forms.NumberInput(
+                attrs={"id": "post-suspension-br", "required": True}
+            ),
+            "suspension_bl": forms.NumberInput(
+                attrs={"id": "post-suspension-bl", "required": True}
+            ),
+            "initial_fuel": forms.NumberInput(
+                attrs={"id": "post-initial-fuel", "required": True}
+            ),
+            "fuel_decrease_rate": forms.NumberInput(
+                attrs={"id": "post-fuel-decrease-rate", "required": True}
+            ),
+            "initial_oil": forms.NumberInput(
+                attrs={"id": "post-initial-oil", "required": True}
+            ),
+            "oil_decrease_rate": forms.NumberInput(
+                attrs={"id": "post-oil-decrease-rate", "required": True}
+            ),
+            "created_at": forms.DateTimeInput(
+                attrs={"id": "post-created-at", "required": True}
+            ),
+        }
+
+
+# User story: #95 Continuous Submission for simulator UI
+# end
```

Comment:
That's standard practice to represent changes for a block of code

Reply:
thank you git blame might help, removed comments

Reply:
Thanks - I was going to say that I disagree. After working in software development for years in large organizations, you would never see comments like this describing changes. It should be part of the version control/history (git metadata), and not part of the code. You should describe this in the commit information, not in the code itself. It is a bad practice that makes code harder to read because it doesn't really add much value and increases the length of the code. Interested parties and developers can look at the relevant commit for that line and see all of the changes.